content/en/docs/languages/java/api-components.md

```diff
@@ -0,0 +1,1691 @@
+---
+title: Record Telemetry with API
+linkTitle: Record Telemetry with API
```

Comment:
@jack-berg - eliminate the stutter: there is no need to repeat the title in `linkTitle` when the two are the same.

Given the prose in the introductory paragraph, I was wondering whether the title of this page could be:

```suggestion
title: API overview
```

If you agree, then the file should be renamed accordingly.

Reply:
The benefit of "Record Telemetry with SDK" is that it has a level of symmetry with [Manage Telemetry with SDK](https://opentelemetry.io/docs/languages/java/sdk/). 

If we switch to "API overview", I think the SDK page should be "SDK overview". 

This might also impact the naming of [Configure the SDK](https://opentelemetry.io/docs/languages/java/configuration/). 

I actually don't have opinions about the naming of these pages. I just care about their content being correct, comprehensive, and coherent. The three C's ha.

Reply:
I am fine with both as well, but I have a slight preference for "Record Telemetry with (the) API" as it makes much clearier to an unexperienced reader that this is where the should look for recording/creating/generating/... telemetry.

Reply:
let's stick with `Record Telemetry with API` for now.

Reply:
I pushed a commit to remove the `linkTitle`, given that its the same as the `title`. 

Can we open a new issue to re-discuss the short vs. descriptive titles debate?

Reply:
We might have an issue/discussion open already, and it might have been "settled". @svrnm?

Reply:
Current title works for me.